### PR TITLE
test

### PR DESCRIPTION
**Pull request by pwntester**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+TEST
```
Comment (pwntester) [inline README.md:1]: test comment

Comment (pwntester) [inline README.md:1]: test comment review

**Review by pwntester — COMMENTED** [review]
submit comment

File: README.md [hunk 2]
```diff
@@ -21,6 +27,11 @@ Plug 'pwntester/octo.nvim'
 
 ## Requirements
 
+BAR
+BAR
+BAR
```
Comment (pwntester) [inline README.md:32]: comment from lines 30 to 32

Comment (tami5) [reply README.md:32]: :heart_eyes: :heart_eyes:

File: README.md [hunk 3]
```diff
@@ -11,6 +15,11 @@ Modified description or comments are highlighted in the signcolumn.
 ### Pull Request (review threads)
 ![](https://camo.githubusercontent.com/97aaf7efe7c8ff45cbc4359f28339fd9f9dd7ba3609fbd14b0649a979af15431/68747470733a2f2f692e696d6775722e636f6d2f71495a5a6b48342e706e67) 
 
+FOO
+FOO
```
Comment (pwntester) [inline README.md:19]: comment on a line to be removed

**Pull request merged**